docgen/src/conditional-display.md

```diff
@@ -6,15 +6,24 @@ nav_groups:
 nav_sort: 1
 ---
 
-Sometimes, you will need to render some widgets instead of another if the query hits no results or if there is currently
-no query used. 
+Sometimes, you will need to render some widgets or content instead of another. To do that you can use the `createConnector` 
```

Comment:
"Using our connector and createConnector (link) approach, you can conditionnaly display content
based on the search state." should be enough

Direct style.